public/video-ui/src/services/WorkflowApi.js

```diff
@@ -1,6 +1,7 @@
 import { pandaReqwest } from './pandaReqwest';
 import { getStore } from '../util/storeAccessor';
 import getProductionOffice from '../util/getProductionOffice';
+// import VideoUtils from '../util/video';
```

Comment:
Why is this commented out?

Reply:
the new stuff related to the scheduledLaunch in this file will have to be commented out until we get rid of scheduling in the workflow component